### DISCUSSION TITLE
[Feature request] Provide Position option for OverlayPanel

### DISCUSSION THREAD

**Discussion by lufo88ita** [primefaces/community#1696] [PrimeNG] [answered]
Hi all,
At the moment OverlayPanel can appear below or above the target element.
It would be great if we can set the position so it can appear on the left/right of the target element.

I found a similar request in this old issue
https://github.com/primefaces/primeng/issues/448
But now it is closed

regards,

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we'll re-evaluate the request in the future (v18)
